pagopa/io-app
# chore(IT Wallet): [SIW-1018] Add discovery info activation screen

> [!WARNING]
> This PR depends on #5688 

## Short description
This PR adds the info screen with some details before to start the DIW activation.

https://github.com/pagopa/io-app/assets/49342144/ae489364-9236-489c-9793-d9d474996800

## List of changes proposed in this pull request
- Add ItwDiscoveryInfoScreen
- Add ItwNavigatorStack

## How to test
Using `io-dev-server-api` enable IT Wallet toggle in user profile screen and tap IT Wallet item under Playground section. Tap on Discovery list item and the discovery info screen should appears.

NOTE: in tos and privacy section the link not works because not yet defined
